packages/reactotron-app/App/Stores/SessionStore.js

```diff
@@ -179,6 +180,15 @@ class Session {
 
       this.commandsManager.all.clear()
       this.commandsManager.all.push(...newCommands)
+    } else if (command.type === 'customCommand.register') {
```

Comment:
We might want to declutter this method a little bit as we get more and more "built in" commands. Maybe have a `BuiltInCommandHandler` of some sort that you pass the command and it just returns `true/false` if we should add the command on to the commandsManager